docs/auth/bearerToken.md

```diff
@@ -0,0 +1,104 @@
+## Getting started with Bearer Token authentication
+
+Bearer Token authentication is one of the most popular forms of authentication and is widely used because of it's percieved security. This guide will walk through how to implement bearer token authentication in Glee.
```

Comment:
```suggestion
Bearer Token authentication is one of the most popular forms of authentication and is widely used because of its percieved security. This guide will walk through how to implement bearer token authentication in Glee.
```